CHANGELOG.md

```diff
@@ -2,10 +2,11 @@
 
 ### Version 0.11.0
 
+* `WikiPage::getSection()` now includes subsections by default; disabling the new $includeSubsections option reverts to the old behavior of returning only the text until the first subsection ([#55])
 * Supports a section name (in addition to an index) in `WikiPage::setText()` and `WikiPage::setSection()` ([#45])
-* Improved section processing in `WikiPage::getText()` ([#33], [#37], [#50])
+* Improves section processing in `WikiPage::getText()` ([#33], [#37], [#50])
```

Comment:
We're mixing present and past tense there, and imperative present in the bug fix entries. Mixing tenses is generally a bad idea. I used imperative due to [this recommendation](http://stackoverflow.com/a/3580764), but there are [opposing](http://stackoverflow.com/a/8059167) [opinions](http://softwareengineering.stackexchange.com/questions/157590/why-is-it-preferred-to-write-a-commit-message-in-present-tense-imperative-mood) too. For some entries, like this new PR, present tense ("now includes", "reverts") seems to work best, hence I made the others consistent. For some older entries, past tense ("Supported optional domain") wouldn't work, as that sounds like the project ceased to support it.

I'm okay with past tense (used that in most of my projects), but then some entries will need further rewriting. While we're at it, the number of 0.11.0 entries has grown somewhat large, so maybe we can group and prefix them according to [A good change log sticks to these principles](http://keepachangelog.com/en/0.3.0/)?